app/routes/events/components/EventAttendeeStatistics.tsx

```diff
@@ -49,14 +49,20 @@ const PieChartWithLabel = ({
 }) => {
   return (
     <>
```

Comment:
You don't need these empty tags now that the content is wrapped in a card component.
```suggestion
```